rs/tests/nested/src/util.rs

```diff
@@ -209,16 +209,19 @@ pub(crate) async fn update_nodes_hostos_version(
     vote_execute_proposal_assert_executed(&governance_canister, proposal_id).await;
 }
 
-pub(crate) fn setup_nested_vm(env: TestEnv, name: &str) {
+pub(crate) fn setup_nested_vm(env: TestEnv, names: &[&str]) {
```

Comment:
Considering this function now sets up multiple VMs what renaming it to:
```suggestion
pub(crate) fn setup_nested_vms(env: TestEnv, names: &[&str]) {
```